Deepgram/Billing/IBillingClient.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Threading.Tasks;
+
+namespace Deepgram.Billing
+{
+    public interface IBillingClient
+    {
+        /// <summary>
+        /// Generates a list of outstanding balances for the specified project. To see balances, the authenticated account must be a project owner or administrator
+        /// </summary>
+        /// <param name="projectId">Unique identifier of the project for which you want to retrieve outstanding balances</param>
+        /// <returns>List of Deepgram balances</returns>
+        Task<Billing[]> GetAllBalancesAsync(string projectId);
```

Comment:
Will need to update based on the BillingClient feedback.

Reply:
Thank you for your detailed comment. I've added a BillingList-Object as you mentioned.